src/strings/PowerFxResources.en-US.resx

```diff
@@ -4115,6 +4115,10 @@
     <value>Named formula must be an expression.</value>
     <comment>This error message shows up when Named formula is not an expression. For example, a = ;</comment>
   </data>
+  <data name="ErrNamedType_MissingTypeLiteral" xml:space="preserve">
+    <value>Named type must be a TypeLiteral.</value>
```

Comment:
>TypeLiteral [](http://example.com/codeflow?start=32&length=11)

AFAIK we don't have a concept of "TypeLiteral" that is exposed to makers, so we should use the common form `type literal`